sklearn/linear_model/_base.py

```diff
@@ -766,8 +767,10 @@ def _check_precomputed_gram_matrix(
     X_scale : ndarray of shape (n_features,)
         Array of feature scale factors used to normalize design matrix.
 
-    rtol : float, default=1e-7
-        Relative tolerance; see numpy.allclose.
+    rtol : float, default=None
+        Relative tolerance; see numpy.allclose and
+        sklearn.utils._testing.assert_allclose.
+        If None, it is set based on the provided arrays' dtypes.
```

Comment:
Please explicit on the number. Before it was clear it was 1e-7

Reply:
I adapted the PR accordingly (see https://github.com/scikit-learn/scikit-learn/pull/22059/commits/9d2ae611bbe8e96fa22bdc406a6939957a5464f7) and now explicitly state the `rtol` values in the docstring